woocommerce/woocommerce-android
# Fix payment selection landscape mode

Closes: #6933 


### Description


It fixes the missing WCPay, checkmark and card holding icons. 
It also decreases the space between the Confirm Payment button in landscape mode
context: [p5T066-3oX-p2](p5T066-3oX-p2#comment-12931)

### Testing instructions


1. Log into a US-based site with both Stripe and WCPay installed
2. Go to the Menu
3. IPP settings
4. Test the Payment selection screen in landscape mode
5. make sure the WCPay, checkmark and the card holding hand icon are displayed
6. make sure the space between the confirm payment button is not too big and does not require a lot of scrolling
7. test the screen on other devices 


### Images/gif


<img src="https://user-images.githubusercontent.com/30724184/178959588-7bf02376-3d69-4d21-b91f-4ea97a0c0ff6.gif" width="350"/> 

## Pixel 2 API 30
<img src="https://user-images.githubusercontent.com/30724184/178959474-4d7a8343-a54a-430a-92be-954a9391c32f.png" width="350"/> 

## Pixel 4
<img src="https://user-images.githubusercontent.com/30724184/178959514-6838ff25-ec7c-4e2b-8d78-25f5c14b6cfd.png" width="350"/>
